config/routes.rb

```diff
@@ -22,6 +22,7 @@
 
   namespace :admin do
     get "/dashboard", to: "base#dashboard"
+    resources :categories
```

Comment:
Maybe we want to limit this to needed routes? From the categories controller, it looks like you are only hitting index and create?